substrate/frame/system/src/lib.rs

```diff
@@ -550,6 +554,8 @@ pub mod pallet {
 
 		type SystemWeightInfo: WeightInfo;
```

Comment:
This also has no comment 🤔

Reply:
Fixed. Also added one to the line above it.